packages/financial-templates-lib/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Arbitrage automation and libraries for UMA financial templates",
   "dependencies": {
     "@ethersproject/bignumber": "^5.4.2",
-    "@google-cloud/logging-winston": "^4.0.4",
```

Comment:
this is unrelated to the main change of this PR. I realized this fixes the issue we've been having for a while in production where this log is generated every time the hub or spoke runs:

![image](https://user-images.githubusercontent.com/12886084/145979840-f0753faa-0b3a-486e-9727-dc9631fd62df.png)